.gitignore

```diff
@@ -1,4 +1,3 @@
 node_modules
 .DS_Store
 .docusaurus
-build
```

Comment:
build/

(don't want to commit build files)

Reply:
yeah you are right, I will rename build.md